examples/where/dapp/src/Calculate.sol

```diff
@@ -11,9 +11,12 @@ contract Calculate {
         require(y == 0 || (z = x * y) / y == x);
     }
 
-    function run(uint a, uint b) public pure returns (uint e) {
+    uint result = 0;
```

Comment:
Sanity check (mine): this would be 0 even without the assignment, right?

Reply:
I believe that is the case yes, but I think it's normally good to be explicit.